hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -367,12 +367,14 @@
       uvm_test_seq: chip_sw_sleep_pin_mio_dio_val_vseq
       sw_images: ["//sw/device/tests/sim_dv:sleep_pin_mio_dio_val_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+use_otp_image=LcStProd"]
```

Comment:
do these need to specifically be in prod..? or is it a.."can be any state" kind of situation? (I wonder if we should randomize that too now..)

Reply:
I chose prod to get around tap and dft straps (and hence, the jtag interface) which are hooked on to the muxed ios and that interferes with this test. Since the purpose of this test is to verify the MIO / DIO signaling in deep sleep, I thought this was ok. Am I missing any obvious issues with this change?

Reply:
Ah you are referring to the continuous sampling and TAP muxing mechanism in DFT enabled LC states?
If we are not testing any life-cycle specific functionality in this test, it should be ok to use PROD so that the continuous sampling mechanism does not interfere with this test.

Reply:
Yes that is right. I did not want continuous strap sampling logic to interfere with the tests that need full access to all DIO/MIO pins. If we disconnect the strap interfaces from the strap pins (and jtag pins), the test bombs due to X prop from the strap sampling logic.

Reply:
sounds good.